spring-cloud-gcp-starters/spring-cloud-gcp-starter-trace/README.adoc

```diff
@@ -0,0 +1,50 @@
+= Google Cloud Stackdriver Trace Spring Boot Starter
+
+Maven coordinates, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-starter-trace</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-starter-trace', version: '1.0.0.BUILD-SNAPSHOT'
+}
+----
+
+This starter uses Spring Cloud Sleuth and provides a `SpanReporter` that sends the trace to Stackdriver Trace.
```

Comment:
Which trace?

Reply:
Will clarify here - it's Sleuth's trace information